### PR TITLE
Basic payment type test passing

### PR DESCRIPTION
**Pull request by elizawmeeks**
## REFERENCE

Ticket #3 

[ERD](https://github.com/Phat-Taupe-Pests/Bangazon) 

## DESCRIPTION
Wrote basic test for Payment Type and basic implementation so it will pass.

## STEPS TO TEST
1. Clone this branch to a local repo
3. Go into Bangazon.Tests and run `dotnet test`

### CODE REVIEW

File: Bangazon.Tests/Bangazon_PaymentTypeManagerShould.cs [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+            
+            int paymentTypeID = _ptm.AddNewPaymentType(newPaymentType);
+
+            Assert.True(paymentTypeID !=0);
+        }
+
+        // Burns the database down because the paint color is wrong.
+            public void Dispose()
+        {
+            _db.Delete("DELETE FROM customer");
```
Comment (BSGreaves) [inline Bangazon.Tests/Bangazon_PaymentTypeManagerShould.cs]: Should be DELETE FROM paymentType

File: Bangazon/Models/PaymentType.cs [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+namespace BangazonCLI
+{
+    // Gets and sets public properties of a payment type.
+    public class PaymentType
+    {
+        public int paymentTypeID {get;}
+        public int customerID {get; set;}
+        public int accountNumber {get; set;}
+        public string name {get; set;}
+        public Customer customer {get; set;}
```
Comment (BSGreaves) [inline Bangazon/Models/PaymentType.cs]: Is this necessary? I can't think of an instance in which we'd need to call on the Customer FROM the payment type. Usually we'd take the customer ID, get all their payment types from SQL, then do whatever we need with the payment type (like add it to an order).

**Comment by BSGreaves** [comment]
Made two comments on the files

File: Bangazon/Models/PaymentType.cs (hunk 2 above)
Comment (BSGreaves) [reply Bangazon/Models/PaymentType.cs]: This is RE: line 14

**Comment by augsismyburger** [comment]
The test passed. On board with Ben, however.

File: Bangazon.Tests/Bangazon_PaymentTypeManagerShould.cs [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+    {
+        private readonly PaymentTypeManager _ptm;
+        private readonly dbUtilities _db;
+        // Creates a customer manager and connection with the database..
+        
+        public PaymentTypeManagerShould ()
+        {
+            _db = new dbUtilities("BANGAZONCLI_TEST_DB");
+            _ptm = new PaymentTypeManager(_db);
+            _db.CheckCustomer();
```
Comment (adamreidelbach) [inline Bangazon.Tests/Bangazon_PaymentTypeManagerShould.cs]: Needed to comment this out in order for the test to pass

**Comment by augsismyburger** [comment]
Even Better everything looks good. 👍

**Comment by BSGreaves** [comment]
All tests passed 👍

**Comment by chazhenricks** [comment]
Tests be passssssssssssssssssinallday

**Pull request merged**